src/plugins/simulator/simulations/terrarium/terrarium.scss

```diff
@@ -0,0 +1,54 @@
+// From https://fonts.googleapis.com/css?family=Saira
+@font-face {
+  font-family: 'Saira';
+  font-style: normal;
+  font-weight: 400;
+  font-stretch: 100%;
+  src: url(https://fonts.gstatic.com/s/saira/v19/memWYa2wxmKQyPMrZX79wwYZQMhsyuShhKMjjbU9uXuA71rCks8xliQjQVcuxA.woff2) format('woff2');
```

Comment:
It looks like in other instances we use `url()` to load local assets, like: 
 
```scss
 background-image: url('./assets/arrow-back-icon.svg');
```

Reply:
I'm honestly not sure. I believe this is effectively what happens when you include a google font using a `link` tag (see `src/index.html` for example), which I think is the only way CLUE currently imports fonts. I went with this approach because it's quick and there weren't obvious alternatives. I think that with this approach the font will only be downloaded when the simulator tile is included in the unit, which is an advantage, but including the font in the repo might have the same advantage.